app/controllers/items_controller.rb

```diff
@@ -22,9 +22,18 @@ def create
   def destroy
     @user = User.find(params[:user_id])
     @item = Item.find(params[:id])
-    @item.destroy
-    flash[:notice] = "Item was removed from your list - Congratulations!"
-    redirect_to user_path(@user)
+
+    if @item.destroy
+      flash[:notice] = "Item was removed from your list - Congratulations!"
```

Comment:
what if you store these strings to instance variables and then try to render the instance variables in the js file?  I'm not sure if the flash hash is accessible in the js file.  (that's not a hint...I'm honestly not sure)